config/global.php

```diff
@@ -66,4 +66,10 @@
 
     'Piwik\EventDispatcher' => DI\object()->constructorParameter('observers', DI\get('observers.global')),
 
+    'Zend_Validate_EmailAddress' => function () {
+        return new \Zend_Validate_EmailAddress(array(
+            'hostname' => new \Zend_Validate_Hostname(array(
+                'tld' => false,
```

Comment:
are there other settings we could disable from email validation, to try and accept as many 'valid' emails as possible?